orchestration-shared/src/test/java/uk/gov/di/orchestration/shared/entity/VectorOfTrustTest.java

```diff
@@ -79,6 +79,65 @@ void shouldReturnOnlyLevelsOfConfidenceWhenRequested() {
         assertThat(levelsOfConfidence, equalTo(List.of("P2", "P1")));
     }
 
+    @ParameterizedTest
+    @MethodSource("vtrListsWithLowestCredentialTrustVtrs")
+    void shouldReturnCredentialTrustLevelOfLowestVot(
+            List<VectorOfTrust> vtrList, CredentialTrustLevel expected) {
+        var lowestCredentialLevel = VectorOfTrust.getLowestCredentialTrustLevel(vtrList);
+        assertThat(lowestCredentialLevel, equalTo(expected));
+    }
+
+    public static Stream<Arguments> vtrListsWithLowestCredentialTrustVtrs() {
+        return Stream.of(
+                Arguments.of(
+                        List.of(
+                                VectorOfTrust.of(MEDIUM_LEVEL, LevelOfConfidence.MEDIUM_LEVEL),
+                                VectorOfTrust.of(LOW_LEVEL, LevelOfConfidence.LOW_LEVEL)),
+                        LOW_LEVEL),
+                Arguments.of(
+                        List.of(
+                                VectorOfTrust.of(LOW_LEVEL, LevelOfConfidence.LOW_LEVEL),
+                                VectorOfTrust.of(MEDIUM_LEVEL, null)),
+                        MEDIUM_LEVEL),
```

Comment:
I find this surprising, but I guess it doesn't really matter if this combination is going to be invalid?

Reply:
Yeah, this is due to fail when the logic is implemented to catch mixtures of identity and non-identity lists